woocommerce/woocommerce-gateway-stripe
# Use `hide_gateways_on_settings_page()` to hide gateways from WooCommerce > Settings > Payment instead of CSS

In https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2934 we added CSS to hide our legacy Stripe payment methods from the **WooCommerce > Settings > Payments** list:

**Before:**
![image](https://github.com/woocommerce/woocommerce-gateway-stripe/assets/2275145/dbdb1a73-a52c-456b-ad99-333ae674bd65)

**After**:
![image](https://github.com/woocommerce/woocommerce-gateway-stripe/assets/2275145/5dce925b-186a-4dfd-b38d-ae546db7570d)

In this PR, I'm replacing the CSS approach with using our existing `WC_Stripe_Settings_Controller::hide_gateways_on_settings_page()` function.

### Testing instructions:

1. Set your store currency to EUR so that there are multiple Stripe methods available
2. With UPE enabled.
3. Visit **WooCommerce > Settings > Payments** and confirm only "Stripe" is showing up in the list of payment gateways
4. Toggle UPE turned off.
5. Check **WooCommerce > Settings > Payments** and confirm that only the one Stripe payment method is visible.
